mavis/test/pages/programmes.py

```diff
@@ -13,11 +13,11 @@ def __init__(self, page: Page, test_data: TestData):
         self.test_data = test_data
 
         self.page = page
-
-        self.programme_links = {
-            programme: page.get_by_role("link", name=programme)
-            for programme in Programme
-        }
+        self.current_year_programmes_card = (
+            page.get_by_role("heading")
+            .filter(has_text="2025 to 2026")
```

Comment:
At the moment, this is actually the upcoming academic year. It will probably work for these tests, but worth noting at the moment new patients are added to the current academic year (2024 to 2025). In the future this will be changed.